docs/developer/using_omnisci.rst

```diff
@@ -21,4 +21,8 @@ or use it in your code:
 Since OmniSci is run through its native engine Modin itself sets ``MODIN_ENGINE=Native``
 and you might not specify it explicitly.
 
+.. note::
+   If you encounter ``CommandLine Error: Option 'enable-vfe' registered more than once!`` error when using OmniSci,
```

Comment:
Not sure it is a good idea to mention particular option name, i think it can be different in other LLVM versions... More reliable (but unfortunately not a silver bullet anyway) to mention inconsistency in registered CommandLine options

Reply:
That probably makes sense. I would like to hear other opinions on the changes too.

Reply:
Fixed.